index.html

```diff
@@ -79,271 +80,247 @@ <h2 id="connect">LET'S CONNECT</h2>
 
       <section id= "portfolio" class="work_container">
         <div class="work works0">
-          <img src="images/Snapshoot Portfolio .png" alt=" Project snapshoot" />
+          <!-- <img src="images/Snapshoot Portfolio .png" alt=" Project snapshoot" /> -->
           <div class="Project info">
-            <h3 class="project_name">Tonic</h3>
+            <h3 class="project_name"> </h3>
             <ul class="client">
-              <li id="h3">CANOPY</li>
+              <li id="h3"> </li>
               <li class="Counter">
                 <img src="images/Counter.png" alt="" />
               </li>
 
-              <li class="langue">Back end Dev</li>
+              <li class="langue"> </li>
               <li class="Counter">
                 <img src="images/Counter.png" alt=""/>
               </li>
 
-              <li class="year">2015</li>
+              <li class="year"> </li>
             </ul>
             <p class="project_description">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
+           
             </p>
             <ul class="code_language">
-              <li class="html"><a class="ht" href="#">html</a></li>
-              <li class="css"><a class="ht" href="#">css</a></li>
-              <li class="javascript"><a class="ht" href="#">javascript</a></li>
+              <li ><a class="ht html" href="#"> </a></li>
+              <li ><a class="ht css" href="#"> </a></li>
+              <li ><a class="ht javascript" href="#"> </a></li>
```

Comment:
- [ ] Please kindly ensure that the markup for the works/projects section is completely removed so that it can be generated dynamically with javascript from the projects array created.
- [ ] Also, at the moment the modal is not created dynamically and it is only one modal for all projects. Please kindly ensure it is dynamically generated with Javascript when a project button is clicked.

Reply:
Resolved, thank you.

Reply:
Resolved. thank you.